src/vtools-analysis.py

```diff
@@ -112,9 +112,10 @@ def summarize(infile, df, config_dict, debug):
         keys.append(key)
         vals.append(val)
     # add derived values
-    if "pkt_duration_time" in df:
+    if "pkt_duration_time_ms" in df:
```

Comment:
can you provide a reference for this? 

Also, ffmpeg is notorious for having a large version fleet deployed. We want to prefer the new key name, but resort to the old key name if somebody has only that one

Reply:
Yes, I did, please refer to the difference in above picture.
ffprobe doesn't use pkt_duration_time_ms in any version. This is a variable used only in this tool.